apps/api/src/hyp3_api/routes.py

```diff
@@ -68,7 +67,7 @@ def render_ui():
 
 
 @app.errorhandler(404)
-def error404(e):
+def error404(_):
```

Comment:
I'm not a fan of an `_` as an input argument name. Is this standard practice? Could we use something like `unused_arg` instead?

Reply:
I believe it is standard practice, and it gets rid of PyCharm's warning about an unused parameter.